python/pyspark/sql/column.py

```diff
@@ -250,11 +250,39 @@ def __iter__(self):
         raise TypeError("Column is not iterable")
 
     # string methods
+    _rlike_doc = """ Return a Boolean :class:`Column` based on a regex match.\n
```

Comment:
Without the newlines, the description and and params lines are combined into one line. I don't totally understand why.